notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1964 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip install -Uq jax jaxlib matplotlib
</pre><p>This is unnecessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>